pkgs/applications/audio/musescore/darwin.nix

```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
   sourceRoot = "${appName}.app";
 
   src = fetchurl {
-    url =  "https://github.com/musescore/MuseScore/releases/download/v${concatStringsSep "." (take 3 versionComponents)}/MuseScore-${version}.dmg";
+    url =  "https://github.com/musescore/MuseScore/releases/download/v${concatStringsSep "." (take 2 versionComponents)}/MuseScore-${version}.dmg";
     sha256 = "sha256-lHckfhTTrDzaGwlbnZ5w0O1gMPbRmrmgATIGMY517l0=";
```

Comment:
This hash needs to be updated.

Reply:
Yeah but I can’t build the derivation as I don’t have a Mac…

Reply:
Nevermind, I used `nix store prefetch-file`, sorry…